### PR TITLE
Healthchecks

### PR DESCRIPTION
**Pull request by aintgoin2goa**

### CODE REVIEW

File: server/checks/pollerCheck.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+var _createClass = (function () { function defineProperties(target, props) { for (var i = 0; i < props.length; i++) { var descriptor = props[i]; descriptor.enumerable = descriptor.enumerable || false; descriptor.configurable = true; if ('value' in descriptor) descriptor.writable = true; Object.defineProperty(target, descriptor.key, descriptor); } } return function (Constructor, protoProps, staticProps) { if (protoProps) defineProperties(Constructor.prototype, protoProps); if (staticProps) defineProperties(Constructor, staticProps); return Constructor; }; })();
```
Comment (matthew-andrews) [inline server/checks/pollerCheck.js]: that's a long line.

File: test/fixtures/paywall.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  "name" : "paywall-fixture",
+  "description" : "",
+  "checks": [
+    {
+      "url": "http://localhost:5555/__gtg",
+      "name": "test",
+      "type": "poller",
+      "severity": 2,
+      "businessImpact" : "blah",
+      "technicalSummary" : "god knows",
```
Comment (matthew-andrews) [inline test/fixtures/paywall.json]: :laughing:

File: server/checks/pollerCheck.js (hunk 1 above)
Comment (aintgoin2goa) [inline server/checks/pollerCheck.js]: That's the babel output - code is written in ES6 in /src/

Comment (matthew-andrews) [inline server/checks/pollerCheck.js]: oh… do we need to commit the output?

Comment (matthew-andrews) [inline server/checks/pollerCheck.js]: surely we could compile before deploying?

**Pull request merged**